test/js/samples/dev-warning-missing-data-computed/expected.js

```diff
@@ -11,39 +11,37 @@ import {
 	noop,
 	safe_not_equal,
 	set_data_dev,
-	space,
 	text
 } from "svelte/internal";
 
 const file = undefined;
 
 function create_fragment(ctx) {
 	let p;
-	let t0_value = Math.max(0, ctx.foo) + "";
-	let t0;
-	let t1;
-	let t2;
+
+	let t_fn = ctx => `
```

Comment:
I wonder if we should replace consecutive whitespace characters with a single space, both for consistency with the current behaviour and for reduced bytesize